test/e2e/shoot/create_update_delete.go

```diff
@@ -45,6 +62,12 @@ var _ = Describe("Shoot Tests", Label("Shoot"), func() {
 			g.Expect(shootClient.Client().List(ctx, &corev1.NamespaceList{})).To(Succeed())
 		}).Should(Succeed())
 
+		By("Update Shoot")
+		shootupdatetest.RunTest(ctx, &framework.ShootFramework{
```

Comment:
If we are reusing this now in both e2e and testmachinery tests, should we place it in some common package instead of in `test/testmachinery`?

Reply:
WDYT about [ef9116f](https://github.com/gardener/gardener/pull/6166/commits/ef9116f1b4ddf849fcec710c2a431220cb6a2b40)? Any other suggestion?